generate-docker-bake.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 GOCRONVER="v0.0.10"
 MAIN_TAG="15"
-TAGS_EXTRA="14 13 12"
+TAGS_EXTRA="15 14 13 12"
```

Comment:
Already present in the main tag, this shouldn't be needed, did you generate the hcl with this changes?

Reply:
Fixed this and then generated the hcl